clear_costmap_recovery/src/clear_costmap_recovery.cpp

```diff
@@ -151,6 +154,8 @@ void ClearCostmapRecovery::clear(costmap_2d::Costmap2DROS* costmap){
 
     if(clearable_layers_.count(name)!=0){
 
+      cleared_layers.insert(name);
```

Comment:
Q/F: shouldn't this be at the end, after `clearMap(costmap, x, y);` ? only then it is actually cleared

Reply:
No, because we want to print a warning of layers that weren't cleared due to bad configuration. If the layer fails to clear, it will print an error already (e.g. can't cast to CostmapLayer). Maybe it would be better to rename `cleared_layers` to `found_layers` or something like that? Any suggestions?